inputs/input_dfm.yaml

```diff
@@ -13,6 +13,8 @@ preprocessing:
   # removing both the leading and the trailing empty characters
   strip: True
   only_latin_letters: False
+  # Strings in the inputs will be prefix + string + suffix
+  prefix_suffix: ["|", "|"]
```

Comment:
It can also be empty, i.e. ["", ""], so we will get the same results as before. However, @mcollardanuy wanted to test if adding `start` and `end` tokens can help (particularly when we have a capital letter in the beginning)

Reply:
It really does, given the initial results we saw